tests/factories/item_factories.py

```diff
@@ -14,10 +14,4 @@ class ItemFactory(SQLAlchemyModelFactory):
     author = 'Scott Gladstone'
     summary = 'Summary of the feed item.'
     text = 'Text of the feed item.'
-    feed = factory.SubFactory(FeedFactory)
-
-class EbolaItemFactory(ItemFactory):
```

Comment:
There's still only one test for it, so I don't think it's necessary for now. If we end up determining that we want multiple tests using EbolaItem, we'll add it back later.